src/handlers/itemStore.ts

```diff
@@ -121,5 +121,90 @@ export const itemStoreHandlers: NatsHandler[] = [
                 }
             }
         }
-    ]
+    ],
+    [
+        "item_store_inventory_add_item",
+        async (subscription: Subscription): Promise<void> => {
+            const client = getClient();
+
+            for await (const message of subscription) {
+                const inventory_item = jsonCodec.decode(message.data) as JSONInventoryItem;
+
+                try {
+                    const reponse = await client.index({
+                        index: ELASTICSEARCH_INVENTORY_INDEX_NAME, 
+                        id: String(inventory_item.user_id),
```

Comment:
as discussed, the id can be genrated automatically by elasticsearch so we just need to remove this line